### DISCUSSION TITLE
About Table

### DISCUSSION THREAD

**Discussion by hato-maasaa** [heroui-inc/heroui#1151] [Feedback]
I am Japanese. I became interested in Next UI and have been using it. While using the Table component, I noticed that Table.Cell does not allow line breaks for text. Personally, I believe that leaving the decision to include line breaks to the user would make the component more versatile.

**Comment by emptyfist** [comment]
Hi there.
I think you can break line in Table.Cell use override styles.
Please take a look following articles.

https://nextui.org/docs/theme/override-styles
https://nextui.org/docs/components/table
![image](https://github.com/nextui-org/nextui/assets/90226717/4ed8f118-e229-4725-b87a-56e95860c726)
